core/frontend/src/EmphasizeElementsProps.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Rendering
+ */
+
+import { AppearanceOverrideProps as CommonAppearanceOverrideProps, EmphasizeElementsProps as CommonEmphasizeElementsProps, FeatureOverrideType as CommonFeatureOverrideType } from "@bentley/imodeljs-common";

```

Comment:
Just curious/nit picking here. Did you need to alias the imports to re-export them?

ie, could you have done this?
```
import {  FeatureOverrideType } from "@bentley/imodeljs-common";
/** Options for overriding element appearance.
 * @deprecated use [FeatureOverrideType]($common)
 * @public
 */
export FeatureOverrideType;
```

Reply:
export FeatureOverrideType; <- not valid.

Reply:
He's saying: don't do import `as CommonXyz` you could just import and export `Xyz` directly.

Reply:
Yeah, I get that...you get stuff like this trying to do that:

Parsing error: Declaration or statement expected.

Reply:
If someone wants to mess with this, disable the auto-merge and go ahead, I'm eating lunch.

Reply:
too late. :)

Reply:
This works, but you lose the ability to deprecate/document them.
```
import { AppearanceOverrideProps, EmphasizeElementProps, FeatureOverrideType } from "@bentley/imodeljs-common";
export { AppearanceOverrideProps, EmphasizeElementProps, FeatureOverrideType };
```

Reply:
Yes, likely also triggers the error about exports w/o release tags...